tests/unittests/BackendTest.cpp

```diff
@@ -53,23 +55,27 @@ TEST(Interpreter, profileQuantizationForANetwork) {
   Node *O = F->createFullyConnected(bindings, "fc", A, 4);
   O = F->createRELU("relu", O);
   O = F->createRegression("reg", O, Ex);
+  F->createSave("ret", O);
```

Comment:
just curious: why does the HM mean we need a save here? does the partitioner assume that all graphs have one?

Reply:
So this could have been split into a separate PR, turns out without the Save node DCE wipes the whole function...